PeakLocalisations/Localisation/Features/AddMetrics.xcstrings

```diff
@@ -49,6 +61,12 @@
             "state" : "translated",
             "value" : "You’ve added everything!"
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "¡Han añadido todo!"
```

Comment:
Has instead of Han, to mean you are speaking to a singular user and not more people

Reply:
Or rather `Ha` right? I'm using the formal way everywhere, or at least that's the goal

Reply:
Yes, Ha for formal